### PR TITLE
feat: upgrade kroma version

### PR DESCRIPTION
**Pull request by ccrayz**
Changed the executable docker compose structure and the contents of REAMD.md to match the Kroma v0.3.0 version change.

### CODE REVIEW

**Review by wo-o — APPROVED** [review]
LGTM

File: .env.sample [hunk 1]
```diff
@@ -1,29 +1,63 @@
...
 KROMA_VALIDATOR__PRIVATE_KEY=
 
-# TBD
-KROMA_VALIDATOR__CHALLENGER_DISABLED=true
 KROMA_VALIDATOR__GUARDIAN_ENABLED=false
-KROMA_VALIDATOR__PROVER_GRPC=
+
+# Enable the challenger feature of kroma-validator.
+# To use the challenger feature, you need to run an additional kroma-prover.
+KROMA_VALIDATOR__CHALLENGER_ENABLED=false
```
Comment (0xHansLee) [inline .env.sample:36]:
```suggestion
KROMA_VALIDATOR__CHALLENGER_ENABLED=true
```

File: .env.sample [hunk 2]
```diff
@@ -1,29 +1,63 @@
...
+
+# Required if the value of KROMA_VALIDATOR__CHALLENGER_ENABLED is true.
+# The endpoint for the validator to send GRPC requests to the kroma-prover for proof generation.
```
Comment (0xHansLee) [inline .env.sample]:
```suggestion
# The endpoint for the validator to send jsonRPC requests to the kroma-prover for proof generation.
```

File: .env.sample [hunk 3]
```diff
@@ -1,29 +1,63 @@
...
+KROMA_VALIDATOR__PROVER_GRPC=http://kroma-prover-proxy:6000
```
Comment (0xHansLee) [inline .env.sample]:
```suggestion
KROMA_VALIDATOR__PROVER_RPC=http://kroma-prover-proxy:6000
```

**Comment by seolaoh** [comment]
Could you squash two commits to one commit?

File: .env.sample [hunk 4]
```diff
@@ -1,29 +1,63 @@
...
+IMAGE_TAG__KROMA_NODE=v0.3.0
+IMAGE_TAG__KROMA_VALIDATOR=v0.3.0
+IMAGE_TAG__KROMA_PROVER_PROXY=v0.1.1
+IMAGE_TAG__KROMA_PROVER=v0.1.2
 
-# Specifies the L1 RPC ENDPOINT. 
-# By default, you can check and use an rpc endpoint with the following path: https://sepolia.dev/, but you can also use Alchemy or Infura, etc.
+# Specifies the L1 RPC ENDPOINT.
+# By default, you can check and use an rpc endpoint with the following path: https://sepolia.dev/,
+# but you can also use Alchemy or Infura, etc.
```
Comment (seolaoh) [inline .env.sample]:
```suggestion
# or you can also use Alchemy or Infura, etc.
```

File: .env.sample [hunk 5]
```diff
@@ -1,29 +1,63 @@
...
-IMAGE_TAG__KROMA_VALIDATOR=v0.2.2
+IMAGE_TAG__KROMA_GETH=v0.2.1
+IMAGE_TAG__KROMA_NODE=v0.3.0
+IMAGE_TAG__KROMA_VALIDATOR=v0.3.0
+IMAGE_TAG__KROMA_PROVER_PROXY=v0.1.1
+IMAGE_TAG__KROMA_PROVER=v0.1.2
 
-# Specifies the L1 RPC ENDPOINT. 
-# By default, you can check and use an rpc endpoint with the following path: https://sepolia.dev/, but you can also use Alchemy or Infura, etc.
+# Specifies the L1 RPC ENDPOINT.
```
Comment (seolaoh) [inline .env.sample]:
```suggestion
# Specify the L1 RPC ENDPOINT.
```

File: .env.sample [hunk 6]
```diff
@@ -1,29 +1,63 @@
...
+KROMA_NODE__BOOT_NODES=enr:-Jm4QD1ykM_4VBFhKk5e9X-xIv7GRPQgCTDwqkFK2R15R4nhAXqXiq3ZdjEeD8AfdMVeMcHwmQaoen-zAd5g0DbH0FqGAYolxPtigmlkgnY0i2tyb21hLXN0YWNrg7YSAIlzZWNwMjU2azGhAukNH6ZNUapztQET17wn_DRCYIIQJ9Xwb6gCKhIW3O__g3RjcIIjK4N1ZHCCIys,enr:-Jm4QA7fjlcTsjqg4jBNjsYTklSR289dGeNI71mix5WWsexoXqAXJb230Gs-8ylbMWQotiggk7oPlWw6P3i_wjmuuMeGAYolxPyXgmlkgnY0i2tyb21hLXN0YWNrg7YSAIlzZWNwMjU2azGhAmoL9VwTOmp4V7p1adVGk8t-inTnp5BS6BxXXDwO4j7Mg3RjcIIjK4N1ZHCCIys
 
-####################################
-# The following settings are used to act as a validator.
+#===========================================================================#
+#     ↓ The following settings are used to act as a Kroma-Validator ↓       #
+#===========================================================================#
 # We recommend the values we provide as examples by default
```
Comment (seolaoh) [inline .env.sample]:
```suggestion
# We recommend to use the values we provide as examples by default.
```

File: .env.sample [hunk 7]
```diff
@@ -1,29 +1,63 @@
...
+# If you are using MNEMONIC and HD_PATH, enter them here
 KROMA_VALIDATOR__MNEMONIC=
 KROMA_VALIDATOR__HD_PATH=
+
+# If you are using PRIVATE_KEY, enter them here
 KROMA_VALIDATOR__PRIVATE_KEY=
 
-# TBD
-KROMA_VALIDATOR__CHALLENGER_DISABLED=true
 KROMA_VALIDATOR__GUARDIAN_ENABLED=false
```
Comment (seolaoh) [inline .env.sample]: This flag is not required flag, and the value is false by default. User should not change this flag, so remove this line.
```suggestion
```

File: .env.sample (hunk 2 above)
Comment (seolaoh) [inline .env.sample]:
```suggestion
# The endpoint for the validator to send JSON-RPC requests to the kroma-prover for proof generation.
```

File: .env.sample (hunk 3 above)
Comment (seolaoh) [inline .env.sample]:
```suggestion
KROMA_VALIDATOR__PROVER_RPC=http://kroma-prover-proxy:6000
```

File: .env.sample [hunk 8]
```diff
@@ -1,29 +1,63 @@
...
+
+# Required if the value of KROMA_VALIDATOR__CHALLENGER_ENABLED is true.
+# The endpoint for the validator to send GRPC requests to the kroma-prover for proof generation.
+KROMA_VALIDATOR__PROVER_GRPC=http://kroma-prover-proxy:6000
+
+
+#===========================================================================#
+#    ↓ The following settings are used to act as a Kroma-Prover-Proxy ↓     #
+#===========================================================================#
+# We recommend the values we provide as examples by default
```
Comment (seolaoh) [inline .env.sample]:
```suggestion
# We recommend to use the values we provide as examples by default.
```

File: README.md [hunk 9]
```diff
@@ -1,5 +1,15 @@
-# Run a Kroma up
-We provide an execution guide for users to operate Kroma Full Node and participate in the network as a validator.
+# Kroma up
+
+Kroma team thinks it's really important for people to run their own Kroma nodes.
+
+So we provide a repository where you can get a Kroma node up and running with a simple command called kroma-up.
```
Comment (seolaoh) [inline README.md]:
```suggestion
So we provide this repository where you can get a Kroma node and run it with a simple command.
```

File: README.md [hunk 10]
```diff
@@ -8,10 +18,20 @@ There are tools that need to be prepared before getting started
 * Install [Docker](https://docs.docker.com/engine/install/)
 
 ## Recommended Hardware
+
+Indicates the basic hardware resources required to operate a Kroma nodes.
```
Comment (seolaoh) [inline README.md]:
```suggestion
Indicates the basic hardware resources required to operate a Kroma node.
```

File: README.md [hunk 11]
```diff
@@ -1,5 +1,15 @@
...
+
+Kroma team thinks it's really important for people to run their own Kroma nodes.
+
+So we provide a repository where you can get a Kroma node up and running with a simple command called kroma-up.
+
+###  What can you do?
+
+Kroma up allows users to join the network by running a full node or validator.
+
+In addition, we can operate the validator in two modes, output submitter and challenger, for smooth chain ecosystem operation.
```
Comment (seolaoh) [inline README.md]:
```suggestion
In addition, we can operate the validator in two modes, **output submitter** and **challenger**, for smooth chain ecosystem operation.
```

File: README.md [hunk 12]
```diff
@@ -35,20 +55,115 @@ Options are divided into required and optional, and for the optional ones we rec
...
 ```
 docker compose --profile fullnode up -d
 ```
 
-## Step 5. Start validator node
-The command to run a *validator node* is as follows (It is recommended to run it through the following command.)
+## Step 5. Start Validator
+As mentioned above, the validator can run in two modes **output submitter** and **challenger**.
+
+The steps above show you how to run it in each mode.
```
Comment (seolaoh) [inline README.md]:
```suggestion
The steps below show you how to run it in each mode.
```

File: README.md [hunk 13]
```diff
@@ -35,20 +55,115 @@ Options are divided into required and optional, and for the optional ones we rec
...
+
+## 5-1. Run a Output submitter mode
+Turning on **output submitter mode** is meaningful in that it contributes to decentralizing output submissions for security-related withdrawals on the chain, and you will be rewarded for submitting.
```
Comment (seolaoh) [inline README.md]:
```suggestion
Turning on **output submitter mode** is meaningful in that it contributes to decentralize output submissions for security-related withdrawals on the chain, and you will be rewarded for submitting.
```

File: README.md [hunk 14]
```diff
@@ -1,5 +1,16 @@
...
+
+So we provide a repository where you can get a Kroma node up and running with a simple command called kroma-up.
+
+###  What can you do?
+
+Kroma up allows users to join the network by running a full node or validator.
+
+In addition, we can operate the validator in two modes, output submitter and challenger, for smooth chain ecosystem operation.
+
+> A guide to running challenger mode will be available soon.
```
Comment (seolaoh) [inline README.md]:
```suggestion
> A guide to run in challenger mode will be available soon.
```

File: README.md [hunk 15]
```diff
@@ -35,15 +51,19 @@ Options are divided into required and optional, and for the optional ones we rec
...
 ```
 docker compose --profile fullnode up -d
 ```
 
-## Step 5. Start validator node
-The command to run a *validator node* is as follows (It is recommended to run it through the following command.)
+## Step 5. Start Validator
+Validators work in **output submitter mode** by default.
+
+Turning on *output submitter mode* is meaningful in that it contributes to decentralizing output submissions for security-related withdrawals on the chain, and you will be rewarded for submitting.
```
Comment (seolaoh) [inline README.md]:
```suggestion
Turning on *output submitter mode* is meaningful in that it contributes to decentralize output submissions for security-related withdrawals on the chain, and you will be rewarded for submitting.
```

File: envs/sepolia/prover-proxy.env [hunk 16]
```diff
@@ -0,0 +1,10 @@
+JSONRPC_ADDR=0.0.0.0
```
Comment (seolaoh) [inline envs/sepolia/prover-proxy.env:1]: We will add this file and `prover.env` file later.

File: envs/sepolia/validator.env [hunk 17]
```diff
@@ -1,4 +1,5 @@
 VALIDATOR_L1_ETH_RPC=${L1_RPC_ENDPOINT}
+VALIDATOR_L2_ETH_RPC=${L2_RPC_ENDPOINT}
```
Comment (seolaoh) [inline envs/sepolia/validator.env]:
```suggestion
VALIDATOR_L2_ETH_RPC=http://kroma-geth:8551
```

**Review by seolaoh — APPROVED** [review]

**Pull request merged**